server/src/main/java/com/objectcomputing/MemberProfileController.java

```diff
@@ -1,6 +1,20 @@
-package com.objectcomputing ;
+package com.objectcomputing;
 
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+import javax.annotation.Nullable;
+import javax.validation.Valid;
+
+import com.objectcomputing.member.MemberProfile;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.annotation.Body;
 import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.http.annotation.Put;
 
 @Controller("/team-profile")
 public class MemberProfileController {
```

Comment:
Let's go with `/member-profile` if that's what we are calling the controller.  Either that or `/team-member-profile`, but I would rename the classes to match (i.e. `TeamMemberProfileController`) if you decide to do that.